packages/browser/src/profiling/utils.ts

```diff
@@ -446,6 +456,120 @@ function isValidProfile(profile: JSSelfProfile): profile is JSSelfProfile & { pr
   return true;
 }
 
+// Keep a flag value to avoid re-initializing the profiler constructor. If it fails
+// once, it will always fail and this allows us to early return.
+export let PROFILING_CONSTRUCTOR_FAILED: 0 | 1 = 0;
```

Comment:
I don't see us imorting this anywhere. Should we remove the export? Also: Minification will minify booleans to 1/0 anyhow so it should be fine to make this a boolean.

Reply:
100%